packages/core/src/components/button/_button-group.scss

```diff
@@ -147,7 +146,6 @@ Styleguide components.button-group.css
 
       &::after {
         @include pt-button-minimal-divider();
-
```

Comment:
I'm personally a fan of that whitespace, has helped me parse the code easier more than once (I've even added that space in places where it wasn't before). Why _forbid_ it?